dist.sh

```diff
@@ -60,3 +73,14 @@ done
 for f in $core_files; do
     cp $f $dist_dir/
 done
+
+case $platform in
+    windows)
+        rm -f $dist_dir/workcraft
+        7z a -r ${dist_dir}.zip $dist_dir >/dev/null
+        ;;
+    linux)
+        rm -f $dist_dir/workcraft.bat
+        tar -czf ${dist_dir}.tar.gz $dist_dir
```

Comment:
Something like this would do for tar (notice the `C` flag and the extra dirname parameter):
```
tar czfC ${dist_dir}.tar.gz `dirname $dist_dir` `basename $dist_dir
````
I am not sure how to do it for `7z` though... 

Probably a cleaner code would be something like this (not tested):
```
cd `dirname $dist_dir`
base_dir=`basename $dist_dir`
case $platform in
    windows)
        rm -f $base_dir/workcraft
        7z a -r ${base_dir}.zip $base_dir >/dev/null
        ;;
    linux)
        rm -f $base_dir/workcraft.bat
        tar czf ${base_dir}.tar.gz $base_dir
        ;;
esac
```